infra/scripts/seed-roles.sh

```
@@ -0,0 +1,92 @@
+
+# Default values if not set
+DB_HOST=${DB_HOST:-localhost}
+DB_PORT=${DB_PORT:-3306}
+DB_USER=${DB_USER:-root}
+DB_PASSWORD=${DB_PASSWORD:-hotakupassword}
+DB_NAME=${DB_NAME:-hotaku_db}
+
+# Colors for output
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+YELLOW='\033[1;33m'
```

Shellcheck (0.10.0)

[warning] YELLOW appears unused. Verify use (or export if used externally).

(SC2034)